pkg/tsdb/loki/testdata/empty.json

```diff
@@ -1 +1,3 @@
-{}
\ No newline at end of file
+{
```

Comment:
hi @KonH , thanks for working on this issue!

the loki code changes are OK, but there's a little misunderstanding with this test-snapshot-file.

you see, the `empty.json` is the input for the test, and contains data in the loki-response-format,
and the `empty.golden.jsonc` is the snapshot of the output of the test, that we use to compare.

so in this case, i recommend:
1. revert the change to `empty.json`
2. run the unit test at https://github.com/grafana/grafana/blob/main/pkg/tsdb/loki/framing_test.go#L21
3. the unit test will "fail", because the output has slightly changed, and does not match with `empty.golden.jsonc`. also, it  will modify the `empty.golden.jsonc` with the new output.
4. check the `empty.golden.jsonc` with `git diff`, and if the change is reasonable, commit it to the PR (i tried it, and i think it is reasonable)

thanks!

Reply:
@KonH , when you do a `make run`, to start the go-server, does that command work for you?

(i usually just start "make run" (because i use that for other purposes too), and make sure the server is running, then stop it, and then i run the tests)

Reply:
Don't try `make run` before, do it now and now `go run build.go build` and `go test` finished successfully. A bit strange but thanks!